docs/compliance/cis/k8s.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Is inclusion of this premature?

Reply:
Not necessarily ... especially since it's already well-established that EE is coming with Kube in the next release. I can make a note that the next major release of Docker EE will include Kubernetes and that this Benchmark can service as a guide for hardening those Kubernetes components included with the next release of EE.